utils/auto_dockerfile_update/update_dockerfiles.py

```diff
@@ -124,8 +139,9 @@ def update_internal_base_dockerfile(git_repo: Repo) -> None:
         latest_tag_last_updated = latest_tag['last_updated']
         outdated_files = [file for file in dependency_list if
                           is_docker_file_outdated(file, latest_tag_name, latest_tag_last_updated)]
-        for index, batch_slice in enumerate(batch(outdated_files, 25)):
-            branch_name = fr"autoupdate/{base_image}_{index}"
+        for batch_slice in batch(outdated_files, BATCH_SIZE):
+            image_names = reduce(lambda a, b: f"{a}_{b}", [file['name'] for file in batch_slice])
+            branch_name = fr"autoupdate/{base_image}_{latest_tag_name}_{image_names}"
```

Comment:
Would it be better to do:
`autoupdate/{base_image}_{image_names}_{latest_tag_name}`?

Reply:
yeah, I'll also seperate with - not _ to be a bit clearer